tests/e2e/utils/helpers.js

```diff
@@ -0,0 +1,64 @@
+const axios = require('axios')
```

Comment:
rename `helpers.js` -> `apiHelper.js`

Reply:
I think it would be better to place `uploadFile` and `deleteFile` to a new file `fileHelper.js` and keep the remaining code in the same file renamed to `apiHelper.js`, isn't it?